hash/megadriv.xml

```diff
@@ -22665,6 +22845,17 @@ Notice that these are not working on real hardware due to bugged code with VDP i
 		</part>
 	</software>
 
+<!-- Move Putty with second controller -->
```

Comment:
Should this be a usage note? (`<info name="usage" value="..." />`)

Reply:
Changed as suggested.